pkg/appClone/AppCloneService.go

```diff
@@ -138,7 +138,11 @@ func (impl *AppCloneServiceImpl) CloneApp(createReq *bean.CreateAppDTO, context
 	}
 	if !refAppStatus["TEMPLATE"] {
 		impl.logger.Errorw("status not", "TEMPLATE", cloneReq.RefAppId)
-		return nil, nil
+		return app, nil
+	}
+	if !refAppStatus["CHART"] {
```

Comment:
What's the reason to add this?

Reply:
to check if the global deployment template is present or not, if not then return

Reply:
if !refAppStatus["TEMPLATE"] is checking the same thing. isn't it?